src/t8_geometry/t8_geometry_implementations/t8_geometry_examples.cxx

```diff
@@ -113,6 +113,224 @@ t8_geometry_squared_disk::t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreei
   }
 }
 
+/**
+ * Map the faces of an oktaeder to a spherical surface.
+ * \param [in]  cmesh      The cmesh in which the point lies.
+ * \param [in]  gtreeid    The global tree (of the cmesh) in which the reference point is.
+ * \param [in]  ref_coords  Array of \a dimension many entries, specifying a point in [0,1]^dimension.
+ * \param [out] out_coords  The mapped coordinates in physical space of \a ref_coords.
+ */
+void
+t8_geometry_triangulated_spherical_surface::t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid,
+                                                              const double *ref_coords, const size_t num_coords,
+                                                              double *out_coords) const
+{
+  double n[3]; /* Normal vector of the current triangle. */
+  double r[3]; /* Radial vector through one of triangle's corners. */
+  double p[3]; /* Position vector on the triangle plane. */
+
+  t8_locidx_t ltreeid = t8_cmesh_get_local_id (cmesh, gtreeid);
+  double *tree_vertices = t8_cmesh_get_tree_vertices (cmesh, ltreeid);
+
+  n[0] = tree_vertices[0] + tree_vertices[3] + tree_vertices[6];
```

Comment:
We have a functions for such vector-operations. Please use them.

Reply:
Done.